dali/operators/reader/loader/numpy_loader_gpu.cc

```diff
@@ -31,7 +31,7 @@ void NumpyFileWrapperGPU::Reopen() {
 }
 
 void NumpyFileWrapperGPU::ReadHeader(detail::NumpyHeaderCache &cache) {
-  NumpyHeaderMeta header;
+  HeaderMeta header;
```

Comment:
isn't it `numpy::HeaderMeta`?

Reply:
But id *did compile* @.@ I'll check that in a moment...

Reply:
Fixed, thank you!